src/vm/i386/unixstubs.cpp

```diff
@@ -6,7 +6,11 @@
 
 extern "C"
 {
-    void ThrowControlForThread()
+    void ThrowControlForThread(
+#ifdef WIN64EXCEPTIONS        
```

Comment:
I would like to enable them with this merge unless you can see some benefit from doing it later.